hack/parallel.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
You should either add a `-e` here or check that steps like `cluster-build` succeed before you continue to deployment.